ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -950,6 +971,18 @@ open class DrawerController: UIViewController {
     }
 }
 
+// MARK: - DrawerController: Colors
+
+internal extension DrawerController {
+    private static var drawerToken = MSFDrawerTokens()
```

Comment:
This will keep an instance of the MSFDrawerTokens for the entire life of the application, right?
Probably better to create this instance and return the property in the implementations below.